products/pluto/recipe.yml

```diff
@@ -6,7 +6,7 @@ inputs:
   missing_versions_strategy: find_latest
   datasets:
     - name: dcp_councildistricts_wi
-      version: 22c
+      version: 23d
```

Comment:
Just unpin, I think right? So it's the same as all the other dcp boundaries datasets